src/phoneField/index.tsx

```diff
@@ -181,6 +186,20 @@ export default class PhoneField extends PureComponent<PhoneFieldProps, PhoneFiel
     return null
   }
 
+  componentDidMount() {
+    if (this.ref && this.props.focus) {
+      this.ref.current.focus()
+      this.setState({ hasFocus: true })
+    }
+  }
+
+  componentDidUpdate(prevProps: PhoneFieldProps) {
+    if (this.ref && prevProps.focus !== this.props.focus) {
+      this.ref.current.focus()
+      this.setState({ hasFocus: true })
```

Comment:
Remove this line since it will trigger the `onFocus`

Reply:
Done!